server/jetstream_api.go

```diff
@@ -213,29 +305,39 @@ func (s *Server) sendAPIResponse(c *client, subject, reply, request, response st
 	s.sendJetStreamAPIAuditAdvisory(c, subject, request, response)
 }
 
+// JSEnabledResponse is the response to see if JetStream is enabled for this account.
+type JSApiEnabledResponse struct {
+	Error     *ApiError `json:"error,omitempty"`
+	JetStream bool      `json:"jetstream"`
```

Comment:
nit: since this is a JSApiEnabledResponse, and implies it is about JetStream (JS), then I would have had a field "Enabled" instead of "JetStream". The same way you have "Success" for say JSApiStreamTemplateDeleteResponse, and not "DeleteSuccess".

Reply:
That is fair, I can change to Enabled: bool.